### ISSUE TITLE
integrate ofono (use desktop speaker/mic as headset, start and answer calls)

### ISSUE THREAD

**Issue by subjectdenied** [bboozzoo/mconnect#24]
ofono comes with a great dbus api, i have played around a bit and found that it would be doable to use desktop speaker/mic for input/output on android phones with ease over bluetooth (desktop as headset). also ofono dbus-api allows to dial and answer calls at the phone from the desktop. this would basically allow functionality that is known from macOS

**Comment by rodneyrod** [comment]
@subjectdenied Has phone call capability been added by upstream KDE Connect yet?

**Comment by andyholmes** [comment]
Nope, only notification of "events" and sending SMS is supported:

https://github.com/KDE/kdeconnect-kde/tree/master/plugins/telephony
https://github.com/KDE/kdeconnect-android/blob/master/src/org/kde/kdeconnect/Plugins/TelephonyPlugin/TelephonyPlugin.java
